trino/trino_test.go

```diff
@@ -275,6 +275,52 @@ func TestQueryFailure(t *testing.T) {
 	}
 }
 
+func TestExec(t *testing.T) {
+	c := &Config{
+		ServerURI:         "http://foobar@localhost:8080",
+		SessionProperties: map[string]string{"query_priority": "1"},
+	}
+	dsn, err := c.FormatDSN()
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	db, err := sql.Open("trino", dsn)
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer db.Close()
```

Comment:
after moving to `integration_test.go` you should be able to replace this with
```
	db := integrationOpen(t)
	defer db.Close()
```

Reply:
Done!